### PR TITLE
Update perf test baseline with every master commit

### PR DESCRIPTION
**Pull request by yf225**
If a commit is merged onto master, we will take its perf test numbers as new baseline for master.

I will add a graph to track the perf test numbers as the next step.

### CODE REVIEW

**Comment by apaszke** [comment]
Do I understand correctly that once this is merged the perf numbers will no longer be checked in to the remote, but will only update some local copy in our CI instances?

File: .jenkins/short-perf-test-cpu.sh [hunk 1]
```diff
@@ -8,10 +8,37 @@ export PATH=/opt/conda/bin:$PATH
 
 echo "Running CPU perf test for PyTorch..."
 
+# Get last master commit hash
+export PYTORCH_COMMIT_ID=$(git log --format="%H" -n 1)
+
+# Get baseline file from https://github.com/yf225/perf-tests
+cp /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json perf_test_numbers_cpu.json
```
Comment (ezyang) [inline .jenkins/short-perf-test-cpu.sh]: Doesn't have to be for this PR, but let's be mindful of people who will want to run these perf test scripts directly when they want to reproduce CI results. Hardcoded paths like this make the script less portable.

**Comment by ezyang** [comment]
It's not going to be a local copy; it'll be an honest to goodness Git repo on GitHub that the CI will push to.

**Review by ezyang — APPROVED** [review]

**Comment by apaszke** [comment]
There’s no push in the script so I wasn’t sure. So it will be a repo parallel to our main one, but with extra perf details?

**Comment by yf225** [comment]
@apaszke The repo is at https://github.com/yf225/perf-tests for now, and we will move it into the PyTorch org when we are ready to do so

**Comment by apaszke** [comment]
Perfect :+1: I just wanted to clarify it's not pytorch/pytorch 😄

**Pull request merged**